src/Interpreters/Session.cpp

```diff
@@ -218,13 +234,12 @@ void NamedSessionData::release()
     parent.releaseSession(*this);
 }
 
-std::optional<NamedSessionsStorage> Session::named_sessions = std::nullopt;
```

Comment:
As I can see you removed using `Session::named_sessions` but you left this field in class.

Reply:
Yes, you're right, I've removed it now.